bin/index-reader.bat

```diff
@@ -0,0 +1,248 @@
+::
+:: Copyright 2019 GridGain Systems, Inc. and Contributors.
+::
+:: Licensed under the GridGain Community Edition License (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+::
+::
+:: Grid command line loader.
```

Comment:
This comment is copy-pasted from `ignite.bat`, it doesn't make sense in the context of index-reader.

How about `Utility for reading index files` or something else describing functionality of this script?

The same is applicable to captions of other three files.

Reply:
fixed